components/overview/WithdrawModal.js

```diff
@@ -588,6 +591,14 @@ const WithdrawModal = withSlideIn(
 					Math.pow(10, networkInfo.decimalPlaces)
 			) {
 				setCalculationDisabled(true);
+			} else if (
+				controllerAccount &&
+				controllerBalances &&
+				totalTransactionFee +
+					apiInstance?.consts.balances.existentialDeposit.toNumber() >
```

Comment:
change this to transactionFee + (apiInstance?.consts.balances.existentialDeposit.toNumber() / 2)